utilities/cli/holohub.py

```diff
@@ -223,6 +228,9 @@ def _create_parser(self) -> argparse.ArgumentParser:
         run.add_argument(
             "--parallel", help="Number of parallel build jobs (e.g. --parallel $(($(nproc)-1)))"
         )
+        run.add_argument(
+            "--pkg-generator", default="DEB", help="Package generator for cpack (default: DEB)"
+        )
```

Comment:
Could you please comment on this addition? Seems like packaging is required only for `./holohub build`